scm/src/gmtb_scm_type_defs.f90

```diff
@@ -69,9 +70,30 @@ module gmtb_scm_type_defs
     logical                           :: sfc_flux_spec !< flag for using specified surface fluxes instead of calling a surface scheme
     integer                           :: sfc_type !< 0: sea surface, 1: land surface, 2: sea-ice surface
     real(kind=dp)                     :: sfc_type_real(1)
+    integer                           :: veg_type !
+    real(kind=dp)                     :: veg_type_real(1)
+    integer                           :: veg_frac !< 0: sea surface, 1: land surface, 2: sea-ice surface
+    real(kind=dp)                     :: veg_frac_real(1)
+    real(kind=dp)                     :: slopetype(1)
+    real(kind=dp)                     :: shdmin(1)
+    real(kind=dp)                     :: shdmax(1)
+    real(kind=dp)                     :: tg3(1)
+    real(kind=dp)                     :: slmsk(1)
+    real(kind=dp)                     :: canopy(1)
+    real(kind=dp)                     :: hice(1)
+    real(kind=dp)                     :: fice(1)
+    real(kind=dp)                     :: tisfc(1)
+    real(kind=dp)                     :: snwdph(1)
+    real(kind=dp)                     :: snoalb(1)
+    real(kind=dp)                     :: sncovr(1)
+    real(kind=dp)                     :: uustar(1)
+    integer                           :: soil_type !< 1-14?
+    real(kind=dp)                     :: soil_type_real(1)
     integer                           :: mom_forcing_type !< 1: "revealed forcing", 2: "horizontal advective forcing", 3: "relaxation forcing"
     integer                           :: thermo_forcing_type !< 1: "revealed forcing", 2: "horizontal advective forcing", 3: "relaxation forcing"
     integer                           :: reference_profile_choice !< 1: McClatchey profile, 2: mid-latitude summer standard atmosphere
+    integer                           :: C_RES !< effective model resolution in cubed sphere resolution (needed for GWD)
+    logical                           :: model_ics !<  true means have land info too
```

Comment:
Note to self: investigate whether this logical is truly needed; existing variables sfc_type and spec_sfc_flux might be sufficient (i.e. if sfc_type == 1 and not spec_sfc_flux, one must expect that surface ICs are provided in the case data file).